<!DOCTYPE html>
<html lang='en'>
<head>
<title>0002-Fix-configure_notify-event.patch - aur.git - AUR Package Repositories</title>
<meta name='generator' content='cgit v1.2.3-56-gcc08'/>
<meta name='robots' content='index, nofollow'/>
<link rel='stylesheet' type='text/css' href='/static/css/cgit.css'/>
<script type='text/javascript' src='/cgit.js'></script>
<link rel='shortcut icon' href='/static/images/favicon.ico'/>
<link rel='alternate' title='Atom feed' href='https://aur.archlinux.org/cgit/aur.git/atom/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git' type='application/atom+xml'/>
<link rel='vcs-git' href='https://aur.archlinux.org/wlroots-hidpi-xprop-git.git/' title='aur.git Git repository'/>
</head>
<body>
		<div id="archnavbar" class="anb-aur">
			<div id="archnavbarlogo"><h1><a href="/" title="Return to the main page">Arch Linux User Repository</a></h1></div>
			<div id="archnavbarmenu">
				<ul id="archnavbarlist">
					<li id="anb-home"><a href="https://www.archlinux.org/" title="Arch news, packages, projects and more">Home</a></li>
					<li id="anb-packages"><a href="https://www.archlinux.org/packages/" title="Arch Package Database">Packages</a></li>
					<li id="anb-forums"><a href="https://bbs.archlinux.org/" title="Community forums">Forums</a></li>
					<li id="anb-wiki"><a href="https://wiki.archlinux.org/" title="Community documentation">Wiki</a></li>
					<li id="anb-bugs"><a href="https://bugs.archlinux.org/" title="Report and track bugs">Bugs</a></li>
					<li id="anb-security"><a href="https://security.archlinux.org/" title="Arch Linux Security Tracker">Security</a></li>
					<li id="anb-aur"><a href="/" title="Arch Linux User Repository">AUR</a></li>
					<li id="anb-download"><a href="https://www.archlinux.org/download/" title="Get Arch Linux">Download</a></li>
				</ul>
			</div>
		</div><!-- #archnavbar -->
<div id='cgit'><table id='header'>
<tr>
<td class='main'>AUR : wlroots-hidpi-xprop-git.git</td></tr>
<tr><td class='sub'>AUR Package Repositories | click <a href='/pkgbase/wlroots-hidpi-xprop-git/'>here</a> to return to the package base details page</td><td class='sub right'></td></tr></table>
<table class='tabs'><tr><td>
<a href='/cgit/aur.git/?h=wlroots-hidpi-xprop-git'>summary</a><a href='/cgit/aur.git/log/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>log</a><a class='active' href='/cgit/aur.git/tree/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>tree</a><a href='/cgit/aur.git/commit/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>commit</a><a href='/cgit/aur.git/diff/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>diff</a><a href='/cgit/aur.git/stats/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>stats</a></td><td class='form'><form class='right' method='get' action='/cgit/aur.git/log/0002-Fix-configure_notify-event.patch'>
<input type='hidden' name='h' value='wlroots-hidpi-xprop-git'/><select name='qt'>
<option value='grep'>log msg</option>
<option value='author'>author</option>
<option value='committer'>committer</option>
<option value='range'>range</option>
</select>
<input class='txt' type='search' size='10' name='q' value=''/>
<input type='submit' value='search'/>
</form>
</td></tr></table>
<div class='path'>path: <a href='/cgit/aur.git/tree/?h=wlroots-hidpi-xprop-git'>root</a>/<a href='/cgit/aur.git/tree/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>0002-Fix-configure_notify-event.patch</a></div><div class='content'>blob: 4ffc0bfacb09672d1bcef11af85132fd44750e21 (<a href='/cgit/aur.git/plain/0002-Fix-configure_notify-event.patch?h=wlroots-hidpi-xprop-git'>plain</a>)
<table summary='blob content' class='blob'>
<tr><td class='linenumbers'><pre><a id='n1' href='#n1'>1</a>
<a id='n2' href='#n2'>2</a>
<a id='n3' href='#n3'>3</a>
<a id='n4' href='#n4'>4</a>
<a id='n5' href='#n5'>5</a>
<a id='n6' href='#n6'>6</a>
<a id='n7' href='#n7'>7</a>
<a id='n8' href='#n8'>8</a>
<a id='n9' href='#n9'>9</a>
<a id='n10' href='#n10'>10</a>
<a id='n11' href='#n11'>11</a>
<a id='n12' href='#n12'>12</a>
<a id='n13' href='#n13'>13</a>
<a id='n14' href='#n14'>14</a>
<a id='n15' href='#n15'>15</a>
<a id='n16' href='#n16'>16</a>
<a id='n17' href='#n17'>17</a>
<a id='n18' href='#n18'>18</a>
<a id='n19' href='#n19'>19</a>
<a id='n20' href='#n20'>20</a>
<a id='n21' href='#n21'>21</a>
<a id='n22' href='#n22'>22</a>
<a id='n23' href='#n23'>23</a>
<a id='n24' href='#n24'>24</a>
<a id='n25' href='#n25'>25</a>
<a id='n26' href='#n26'>26</a>
<a id='n27' href='#n27'>27</a>
<a id='n28' href='#n28'>28</a>
<a id='n29' href='#n29'>29</a>
<a id='n30' href='#n30'>30</a>
<a id='n31' href='#n31'>31</a>
</pre></td>
<td class='lines'><pre><code>From 2fbf42a76ebf3959fb95a6c0ea66a6f04b032070 Mon Sep 17 00:00:00 2001
From: q234rty &lt;q23456yuiop&#64;gmail.com&gt;
Date: Tue, 21 Feb 2023 23:56:26 +0800
Subject: [PATCH 2/2] Fix configure_notify event

<span class="hl kwa">---</span>
 xwayland/xwm.c | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/xwayland/xwm.c b/xwayland/xwm.c
index 4f03b07c..f5ceedb1 100644
<span class="hl kwa">--- a/xwayland/xwm.c</span>
<span class="hl kwb">+++ b/xwayland/xwm.c</span>
&#64;&#64; -1806,10 +1806,10 &#64;&#64; void wlr_xwayland_surface_configure(struct wlr_xwayland_surface *xsurface,
 			.response_type = XCB_CONFIGURE_NOTIFY,
 			.event = xsurface-&gt;window_id,
 			.window = xsurface-&gt;window_id,
<span class="hl kwa">-			.x = x,</span>
<span class="hl kwa">-			.y = y,</span>
<span class="hl kwa">-			.width = width,</span>
<span class="hl kwa">-			.height = height,</span>
<span class="hl kwb">+			.x = values[0],</span>
<span class="hl kwb">+			.y = values[1],</span>
<span class="hl kwb">+			.width = values[2],</span>
<span class="hl kwb">+			.height = values[3],</span>
 		};
 
 		xcb_send_event(xwm-&gt;xcb_conn, 0, xsurface-&gt;window_id,
<span class="hl kwa">-- </span>
2.39.2

</code></pre></td></tr></table>
</div> <!-- class=content -->
<div id="footer">
	<p>
		Copyright &copy; 2004-2023 aurweb Development Team &ndash;
		<strong>AUR packages are user produced content. Any use of the provided files is at your own risk.</strong>
	</p>
</div>
</div> <!-- id=cgit -->
</body>
</html>
